packages/gatsby-source-contentful/package.json

```diff
@@ -33,5 +33,8 @@
     "build": "babel src --out-dir . --ignore __tests__",
     "prepublish": "cross-env NODE_ENV=production npm run build",
     "watch": "babel -w src --out-dir . --ignore __tests__"
+  },
+  "optionalDependencies": {
+    "gatsby-plugin-sharp": "^1.6.46"
```

Comment:
If this is set as optional there should be some kind of check in code if it's available before using it. I think this should be just moved to `dependencies`

Reply:
Lets just move it, everybody working with images should have that one anyways.

I wonder if, at this point of time, there is even a scenario where you would not need `gatsby-plugin-sharp`. Move it & we are done here :)